go/cache/lru_cache_test.go

```diff
@@ -121,31 +114,29 @@ func TestCapacityIsObeyed(t *testing.T) {
 	cache.Set("key1", value)
 	cache.Set("key2", value)
 	cache.Set("key3", value)
-	if sz := cache.UsedCapacity(); sz != size {
-		t.Errorf("cache.UsedCapacity() = %v, expected %v", sz, size)
-	}
+	sz := cache.UsedCapacity()
+	assert.Equal(t, size, sz, "cache.UsedCapacity() = %v, expected %v", sz, size)
```

Comment:
No need to report the expected and found values, since `assert.Equal()` already does so.
```suggestion
	assert.Equal(t, size, sz)
```